woocommerce/woocommerce-android
# Irrelevant JITM Banner does not disappear when pull to refresh

Closes: #7960 


### Description

The JITM banner on the MyStore screen does not disappear when pulled to refresh even when the API returns empty JITMs.
This PR fixes this. 

#### Before this PR
1. Add [A4 barcode generator](https://wordpress.org/plugins/a4-barcode-generator/) plugin to your store
2. Open the WooCommerce Android app
3. Ensure you see the JITM banner.
4. De-activate the A4 barcode generator plugin on the admin page
5. Pull to refresh the MyStore screen in the app
6. The banner isn't disappearing.

#### After this PR
1. Add [A4 barcode generator](https://wordpress.org/plugins/a4-barcode-generator/) plugin to your store
2. Open the WooCommerce Android app
3. Ensure you see the JITM banner.
4. De-activate the A4 barcode generator plugin on the admin page
5. Pull to refresh the MyStore screen in the app
6. The banner is disappearing.




### Testing instructions


1. Add [A4 barcode generator](https://wordpress.org/plugins/a4-barcode-generator/) plugin to your store
2. Open the WooCommerce Android app
3. Ensure you see the JITM banner.
4. De-activate the A4 barcode generator plugin on the admin page
5. Pull to refresh the MyStore screen in the app
6. Ensure the banner is disappearing.

### Images/gif


You can see the demo presentation here: p91TBi-a5h-p2. The demo starts at 9:31. The same build is used for the demo.
